CMakeLists.txt

```diff
@@ -80,19 +79,20 @@ target_include_directories(mbed-os
         ${CMAKE_CURRENT_SOURCE_DIR}
 )
 
-# Default build
 add_subdirectory(cmsis)
-add_subdirectory(components)
-add_subdirectory(connectivity)
 add_subdirectory(drivers)
-add_subdirectory(events)
-add_subdirectory(features)
 add_subdirectory(hal)
 add_subdirectory(platform)
 add_subdirectory(rtos)
-add_subdirectory(storage)
 add_subdirectory(targets)
+add_subdirectory(events EXCLUDE_FROM_ALL)
```

Comment:
This line should go under the comment below.

Reply:
fixing now